### ISSUE TITLE
Analysis module re-use

### ISSUE THREAD

**Issue by simonrnss** [AI-SDC/AI-SDC#127]
tl;dr slightly refactor analysis modules to ensure code re-use

Current module here, analyses the AUC p-values:
https://github.com/AI-SDC/AI-SDC/blob/57b92c0ea96a84eae3e26b46cb4cd38dc6e5e26b/aisdc/generate_report.py#L65-L90

But we also have a module that does this (and corrects for multiple testing):
https://github.com/AI-SDC/AI-SDC/blob/57b92c0ea96a84eae3e26b46cb4cd38dc6e5e26b/aisdc/generate_report.py#L154-L204

It would be neater for the first module above, to use the second one to do its AUC analysis. Should be a straightforward switch, with the first module simply creating an instance of the second, getting it to do this particular bit of analysis, and then using its output.

**Comment by jim-smith** [comment]
@yolaj-nhs moving this to sprint 3 - or is it done already?

**Comment by jim-smith** [comment]
@yolaj-nhs nudge

**Comment by yolaj-nhs** [comment]
> @yolaj-nhs nudge

Ah whoops, I didn't see this, sorry!

Closing this ticket as much of it has already been covered implicitly with the work done in other tickets
